specs/features/Tables.md

```diff
@@ -0,0 +1,260 @@
+# Table element for Adaptive Cards
+This is a proposal for a simple Table element. The main goals of this element are:
+- To allow the creation of table layouts in a straightforward way, as opposed to using a stack of ColumnSet elements which leads to a larger (potentially much larger) payload and potential headaches wrt column sizing
+- To allow accessibility software to present table layouts as such, which is currently impossible
+
+This is NOT a proposal for an advanced Table with features such as:
+- Dynamic column sizing
+- Dynamic column reordering
+- Sorting by columns
+- Etc.
+
+## Design
+### Table
+| Name | Type | Required | Description |
+
+| --- | --- | --- | --- |
```

Comment:
Table markdown here is broken. please see [checked here](https://github.com/microsoft/AdaptiveCards/blob/golddove/5630/specs/features/Tables.md)

Reply:
Fixed, thanks!